model_utils/tests/models.py

```diff
@@ -4,25 +4,24 @@
 from django.utils.encoding import python_2_unicode_compatible
 from django.utils.translation import ugettext_lazy as _
 
+from model_utils import Choices
+from model_utils.fields import SplitField, MonitorField, StatusField
+from model_utils.managers import QueryManager, InheritanceManager
 from model_utils.models import (
     SoftDeletableModel,
     StatusModel,
     TimeFramedModel,
     TimeStampedModel,
 )
-from model_utils.tracker import FieldTracker, ModelTracker
-from model_utils.managers import QueryManager, InheritanceManager
-from model_utils.fields import SplitField, MonitorField, StatusField
 from model_utils.tests.fields import MutableField
-from model_utils import Choices
-
+from model_utils.tests.managers import CustomSoftDeleteManager
+from model_utils.tracker import FieldTracker, ModelTracker
 
 
 class InheritanceManagerTestRelated(models.Model):
     pass
 
 
-
```

Comment:
I believe (@carljm correct me if I'm wrong) that Carl prefers/preferred 3 blank lines between classes intentionally, though that stance could've obviously changed.
Were it my choice, I'd omit the whitespace changes that follow subsequently in this file, as it only muddies the diff, but functionally it doesn't matter, and you're the one contributing herefor which we're thankful, so don't go changing it on my say so if Carl doesn't mind it :)